scalafix/rules/src/main/scala/rsc/rules/pretty/SemanticdbPrinter.scala

```diff
@@ -88,7 +88,7 @@ class SemanticdbPrinter(
             str(sym.desc.value)
           } else if (config.better && config.autoimport && fullEnv.lookup(sym.desc.name).isEmpty) {
             addedImportsScope.addImport(sym)
-            str(sym.desc.value)
+            str(fullEnv.lookupPretty(sym.desc.name))
```

Comment:
I think this would go in the `else if` statement? Basically, you don't want to add an import if it was renamed.

Reply:
I hadn't figured out how addedImportsScope works yet actually, as it doesn't end up adding the import in this case anyway.

Reply:
That's because `autoimport` flag is off by default. We might want to have another test input/output file pair which tests `autoimport` until it is usable enough to be on by default.

Reply:
Nope I lied